dvc/command/dag.py

```diff
@@ -0,0 +1,192 @@
+import argparse
+import logging
+
+from dvc.command.base import CmdBase, append_doc_link
+from dvc.exceptions import DvcException
+
+logger = logging.getLogger(__name__)
+
+
+def _show_ascii(G):
+    from dvc.repo.graph import get_pipelines
+    from dvc.dagascii import draw
+
+    pipelines = get_pipelines(G)
+
+    ret = []
+    for pipeline in pipelines:
+        ret.append(draw(pipeline.nodes, pipeline.edges))
+
+    return "\n".join(ret)
+
+
+def _show_dot(G):
+    import io
+    from networkx.drawing.nx_pydot import write_dot
+
+    dot_file = io.StringIO()
+    write_dot(G, dot_file)
+    return dot_file.getvalue()
+
+
+def _show_tree(G):
+    import networkx as nx
+    from treelib import Tree
+    from dvc.repo.graph import get_pipelines
+
+    pipelines = get_pipelines(G)
+
+    def _get_root(g):
+        (root,) = [node for node, degree in g.in_degree() if degree == 0]
+        return root
+
+    ret = []
+    for pipeline in pipelines:
+        if not nx.is_tree(pipeline):
+            continue
+
+        root = _get_root(pipeline)
+
+        tree = Tree()
+        tree.create_node(root, root)
+        for parent, successors in nx.bfs_successors(pipeline, root):
+            for node in successors:
+                tree.create_node(node, node, parent=parent)
+
```

Comment:
Could do better here for showing the whole project. Could totally show dep tree for each final output of the pipeline.